packages/Button/src/Button.styles.js

```diff
@@ -158,11 +160,17 @@ export const sizeStyles = {
     ${stylers.fontSize(-1)};
     min-height: ${stylers.spacer(4)};
     padding: 6.5px ${tokens.spaceLg};
```

Comment:
You _could_, but I actually don't think it's necessary.  The only case it where it will result in a change should be for the two line buttons.  You could compare your local to the version on paprika.highbond.com, maybe send the screenshots to Jonathan and get his approval – but _I_ think the 1.5px difference is not enough to matter to anyone.